code/modules/projectiles/pins.dm

```diff
@@ -10,9 +10,14 @@
 	attack_verb_continuous = list("pokes")
 	attack_verb_simple = list("poke")
 	var/fail_message = "invalid user!"
-	var/selfdestruct = FALSE // Explode when user check is failed.
-	var/force_replace = FALSE // Can forcefully replace other pins.
-	var/pin_removeable = FALSE // Can be replaced by any pin.
+	/// Explode when user check is failed.
+	var/selfdestruct = FALSE
+	/// Can forcefully replace other pins.
+	var/force_replace = FALSE
+	/// Can be replaced by any pin.
+	var/pin_hot_swappable = FALSE
+	///Can be removed from the gun using tools or replaced by a pin with force_replace
+	var/pin_removeable = TRUE
```

Comment:
Is `removeable` a typo or British? Google is telling me it's a typo but is a bit weird about it

Reply:
Looks like a typo, holdover from before the change I'll update it.